scripts/background/TownTuneManager.js

```diff
@@ -6,7 +6,8 @@
 
 function TownTuneManager() {
 
-	var defaultTune = ["C2", "E2", "C2", "G1", "F1", "G1", "B1", "D2", "C2", "zZz", "G1", "zZz", "C2", "-", "-", "zZz"];
+	// var defaultTune = ["C2", "E2", "C2", "G1", "F1", "G1", "B1", "D2", "C2", "zZz", "G1", "zZz", "C2", "-", "-", "zZz"];
```

Comment:
Not a blocker for merging, but I think it's fine to delete any commented old code.